docs/apache-airflow/administration-and-deployment/web-stack.rst

```diff
@@ -0,0 +1,34 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+Web Stack
+=========
+
+Sometimes you will want to deploy the backend and the frontend behind a
+variable url path prefix. To do so you can configure the url :ref:`config:fastapi__base_url`
+for instance set it to ``http://localhost:29091/d12345``. Be sure to reload the
+fastapi server for the config update to take effect. All the APIs routes will
+now be available through that additional ``d12345`` prefix.
+
+Without rebuilding the frontend, XHR requests and static file queries should be
```

Comment:
Even saying "rebuilding" is going to get people concerned here haha. I'd probably just add this into the last line of the previous paragraph.